### PR TITLE
Last review of French translation.

### PR DESCRIPTION
**Pull request by Zenigata**

### CODE REVIEW

**Comment by Zenigata** [comment]
Why do I have the end of print retromat season commits appear here?

**Comment by fiddike** [comment]
Hi @Zenigata , good question. I just pulled all branches into my local repo and looked at them using SourceTree. These commits actually appear twice, with different commit hashes. Maybe somebody rebased. Hmm. However, a diff from current master to your latest commit on this PR works just fine and only shows your changes. => I guess this will be a very simple merge.

Looking forward merging this as soon as one of the french translators, e.g. @Artusamak indicates it's O.K.

File: backend/app/Resources/views/print/index.html.twig [hunk 1]
```diff
@@ -70,15 +70,14 @@
     <div class="content">
```
Comment (Artusamak) [inline backend/app/Resources/views/print/index.html.twig]: I think that the changes in this file should not be here. @fiddike Your opinion is welcome.

File: index.php [hunk 2]
```diff
@@ -268,17 +268,18 @@ function publish_activities_for_keywords(keywords) {
 </div>
```
Comment (Artusamak) [inline index.php]: There are changes in this file that should not exist at all, shouldn't they?

**Comment by Artusamak** [comment]
Review done, @Zenigata thanks for the fixes, good catches. If you open more french PR, feel free to mention me, i'll be highlighted to review the changes faster. :)

**Comment by Zenigata** [comment]
Hi @fiddike , do you want me to create another merge request? What do we do with this one? Thanks for your time :)

**Comment by fiddike** [comment]
Hi @Zenigata, thank you for the reminder. After attending the World Scrum Gathering I was busy catching up with work and family obligations 8-) Just merged your PR, thank you for the fixes!

**Pull request merged**